awx/main/tests/functional/test_inventory_source_injectors.py

```diff
@@ -192,10 +193,16 @@ def create_reference_data(source_dir, env, content):
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('this_kind', CLOUD_PROVIDERS)
-def test_inventory_update_injected_content(this_kind, inventory, fake_credential_factory, mock_me):
+@pytest('this_kind', discover_available_cloud_provider_plugin_names)
+def test_inventory_update_injected_content(this_kind, inventory, fake_credential_factory, self):
     ExecutionEnvironment.objects.create(name='Control Plane EE', managed=True)
     ExecutionEnvironment.objects.create(name='Default Job EE', managed=False)
+    cloud_providers = discover_available_cloud_provider_plugin_names()
+    for this_kind in cloud_providers:
+        with self.subTest(this_kind=this_kind):
```

Comment:
`self.subTest()` is a thing from `unittest`. Specifically, it exists in classes inheriting from `unittest.TestCase`. This project uses `pytest` and so it should follow the `pytest` ways. Instead of trying to stick `unittest` into `pytest` (which is partially possible but is discouraged and limited), add `pytest-subtest` to dependencies, inject whatever fixture it provides and make calls through that fixture.